README.md

```diff
@@ -1 +1,204 @@
-# Budget-App
\ No newline at end of file
+# Budget App
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  <!-- - [🚀 Live Demo](#live-demo) -->
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Setup](#setup)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+
+
+# 📖 Budget App <a name="about-project"></a>
+
+> Budget App is a Ruby on Rails application that allows a user to login to a transaction page where the user can perform actions like, add or delete categories and transactions, etc.
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Here is a list of technologies used in this project.
+
+<details>
+  <summary>Language</summary>
+  <ul>
+    <li><a href="https://rubylang.org/">Ruby</a></li>
+    <li><a href="https://developer.mozilla.org/">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Framework</summary>
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/">Ruby on rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+
+
+### Key Features <a name="key-features"></a>
+
+- **[Fully functional Backend]**
+- **[Fully functional Frontend]**
+- **[Dynamic routing]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Coming soon...]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
```

Comment:
- [ ] As per the requirement for this capstone, you need to deploy your app using Render, Netlify or GitHub Pages and put the link here. Please do this.